examples/todo-list/src/repositories/todo-list.repository.ts

```diff
@@ -49,4 +52,27 @@ export class TodoListRepository extends DefaultCrudRepository<
   public findByTitle(title: string) {
     return this.findOne({where: {title}});
   }
+
+  async find(
+    filter?: Filter<TodoList>,
+    options?: Options,
+  ): Promise<(TodoList & Partial<TodoListLinks>)[]> {
```

Comment:
All properties in `TodoListLinks` are optional. It's redundant to use `Partial<TodoListLinks>`.

Reply:
Make sense. With the new type `TodoListWithLinks`, this can be further simplified as follows:

```suggestion
  ): Promise<TodoListWithLinks[]> {
```